evm/docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+version: "3.8"
```

Comment:
Any way to test locally that this works?

Reply:
this definition is no longer necessary